Detections/MultipleDataSources/Log4J_IPIOC_Dec112021.yaml

```diff
@@ -70,9 +95,9 @@ query:  |
   let CSlogDestIP = CommonSecurityLog | summarize by IPAddress = DestinationIP, Type;
   let CSlogMsgIP = CommonSecurityLog | extend MessageIP = extract(IPRegex, 0, Message) | summarize by IPAddress = MessageIP, Type;
   let DnsIP = DnsEvents | summarize by IPAddress = IPAddresses, Type;
-  // If you have enabled the imDNS and/or imNetworkSession normalization in your workspace, you can uncomment one or both below.  Reference - https://docs.microsoft.com/azure/sentinel/normalization
```

Comment:
- Many unneeded sources to remove in this query. 
- You don't need to add ASIM functions to the ipMatch list. The search we do in the big union is enough. The original is not optimal to say the least.